src/pools/D3MAavePool.sol

```diff
@@ -100,7 +103,9 @@ contract D3MAavePool is ID3MPool {
 
         TokenLike(dai_).approve(pool_, type(uint256).max);
 
-        CanLike(D3mHubLike(hub_).vat()).hope(hub_);
+        hub = hub_;
+        VatLike vat_ = vat = VatLike(D3mHubLike(hub_).vat());
+        vat_.hope(hub_);
```

Comment:
Don't need to store `vat_` on the stack after Solidity 0.8. Can just reference the immutable `vat` on line 108.

Reply:
Oh great, didn't know about it. Modified!